From 9a719a0d2f5f489eeb9351b0ef913c3693147a4f Mon Sep 17 00:00:00 2001
From: Noah Egnatis <noah.egnatis@gmail.com>
Date: Mon, 8 May 2023 11:38:08 -0400
Subject: [PATCH] COMP: gcc-13 compatability

Using gcc-13 ITK does not compile as discussed in this form entry: https://discourse.itk.org/t/building-itk-error-with-algorithim-module/5927/9. Was able to be fixed thanks to @dzenanz with the inclusion of  <cstdint> to itkMathematicalMorphologyEnums.h
---
 .../include/itkMathematicalMorphologyEnums.h                   | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h b/Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h
index 0411b17f8b5..d897d5c1494 100644
--- a/Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h
+++ b/Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h
@@ -20,6 +20,7 @@
 
 #include <iostream>
 #include "ITKMathematicalMorphologyExport.h"
+#include <cstdint>
 
 
 namespace itk
@@ -38,7 +39,7 @@ class MathematicalMorphologyEnums
    * \brief Algorithm or implementation used in the dilation/erosion operations.
    * \ingroup ITKMathematicalMorphology
    */
-  enum class Algorithm : uint8_t
+  enum class Algorithm : std::uint8_t
   {
     BASIC = 0,
     HISTO = 1,
